src/App/Pages/Send/SendAddEditPageViewModel.cs

```diff
@@ -337,6 +337,11 @@ public async Task<bool> SubmitAsync()
 
             UpdateSendData();
 
+            if (NewPassword != null && string.IsNullOrWhiteSpace(NewPassword)) 
```

Comment:
You can remove the bit of `NewPassword != null && `, `string.IsNullOrWhiteSpace()` already handles the `null` condition for you.

Reply:
In this case though, null is the value we are after. I'm just using IsNullOrWhiteSpace() for the empty/whitespace catch. If NewPassword is already null we don't need to null it out.

Reply:
setting `NewPassword` to `null` if it's already `null`... no harm, no foul. The `if` check just seems a little too complicated/confusing perhaps... maybe it's just me 🤷🏻

Reply:
I def see either way. Went ahead and removed it. Less code is usually better code.